frame/staking/src/pallet/impls.rs

```diff
@@ -870,6 +872,15 @@ impl<T: Config> Pallet<T> {
 			DispatchClass::Mandatory,
 		);
 	}
+
+	/// This will return the currently configured History Depth
```

Comment:
my OCD-comments 😁: 

1. punctuation missing. 
2. line wrapping should ideally be 100, not less :p